worker/src/unstract/worker/worker.py

```diff
@@ -213,7 +215,8 @@ def run_container(
         except Exception as e:
             self.logger.error(f"Failed to run docker container: {e}")
             result = {"type": "RESULT", "result": None, "error": str(e)}
-        self._cleanup_container(container)
+        if container:
+            container.cleanup()
```

Comment:
NIT: Is this clean up happeing always after particular image is run inside worker?

Reply:
Yes.